packages/angular_devkit/build_angular/src/webpack/plugins/any-component-style-budget-checker.ts

```diff
@@ -7,10 +7,11 @@
  */
 
 import * as path from 'path';
-import { Compiler } from 'webpack';
+import { Compiler, compilation } from 'webpack';
```

Comment:
You don't need to use the instance of Compilation.

```suggestion
import { Compiler, Compilation } from 'webpack';
```

Reply:
webpack4 doesn't have exported member `Compilation`